changelog.md

```diff
@@ -7,6 +7,7 @@
 * Add: Pre-commit hook + instructions and configure precommit.ci bot (@lwasser, #293)
 * Fix: Setuptools_scm bug when installing stravalib remotely via GitHub (@lwasser, #331)
 * Fix: fix LatLon unmarshal from string type (@oliverkurth, #334)
+* Fix: allows arithmetic and comparison between multiple quantities (jsamoocha, #335)
```

Comment:
oh shoot - @jsamoocha i just realized that some of the commits in my pr to bump include your pydantic fixes. maybe we can sort through that together i haven't been closely tracking which commit specifically belongs to which but i can give it a go in a comment right now and you can proof this for me. it's weird to be only releasing a part of our work!

Reply:
I think this should be ok. Changelog additions in the `pydantic_model` branch will not be there on `master`. I'll take a close look.